modules/bookList.js

```diff
@@ -0,0 +1,49 @@
+export default class BookList {
+  constructor() {
+    this.bookList = [];
+    this.displayInfo();
+  }
+
+  add(title, author) {
+    if (title && author) {
+      const newBook = {
+        title,
+        author,
+      };
+      this.bookList.push(newBook);
+      localStorage.setItem('bookList', JSON.stringify(this.bookList));
+      // this.bookList = JSON.parse(localStorage.getItem('bookList'));
+      this.displayInfo();
+    }
+  }
+
+  // Check localStorage and if there is any book saved in it, we render it on the page.
+  // This method removed all the content first and then add items afterward.
+  displayInfo() {
+    if (JSON.parse(localStorage.getItem('bookList'))) {
+      this.bookList = JSON.parse(localStorage.getItem('bookList'));
+      const bookShelf = document.getElementById('bookShelf');
+      bookShelf.innerHTML = '';
+      this.bookList.forEach((book, index) => {
+        bookShelf.innerHTML += `
+              <div class="book-row">
+                      <p class="title-p">"${book.title}" By: ${book.author}</p>
+                      <button id="${index}" class="removeButton">Remove</button>
+              </div> `;
+      });
+      const removeButtons = document.querySelectorAll('.removeButton');
+      removeButtons.forEach((removeButton) => {
+        removeButton.addEventListener('click', (event) => {
+          const index = parseInt(event.target.id); // eslint-disable-line
```

Comment:
- [ ]  `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.